src/main/java/net/serenitybdd/cucumber/CucumberWithSerenity.java

```diff
@@ -36,33 +37,25 @@ public static void setRuntimeOptions(RuntimeOptions runtimeOptions) {
     public CucumberWithSerenity(Class clazz) throws InitializationError, IOException
     {
         super(clazz);
+        RuntimeOptionsFactory runtimeOptionsFactory = new RuntimeOptionsFactory(clazz);
+        RuntimeOptions runtimeOptions = runtimeOptionsFactory.create();
+        runtimeOptions.getTagFilters().addAll(environmentSpecifiedTags(runtimeOptions.getTagFilters()));
+        RUNTIME_OPTIONS = runtimeOptions;
     }
 
     public static RuntimeOptions currentRuntimeOptions() {
         return RUNTIME_OPTIONS;
     }
 
-    /**
-     * Create the Runtime. Sets the Serenity runtime.
-     */
-    @Override
-    protected Runtime createRuntime(ResourceLoader resourceLoader,
-                                                     ClassLoader classLoader,
-                                                     RuntimeOptions runtimeOptions) throws InitializationError, IOException {
-        runtimeOptions.getTagFilters().addAll(environmentSpecifiedTags(runtimeOptions.getTagFilters()));
-        RUNTIME_OPTIONS = runtimeOptions;
-        return CucumberWithSerenityRuntime.using(resourceLoader, classLoader, runtimeOptions);
-    }
-
-    private Collection<String> environmentSpecifiedTags(List<? extends Object> existingTags) {
+    private static Collection<String> environmentSpecifiedTags(List<? extends Object> existingTags) {
         EnvironmentVariables environmentVariables = Injectors.getInjector().getInstance(EnvironmentVariables.class);
         String tagsExpression = ThucydidesSystemProperty.TAGS.from(environmentVariables,"");
         List<String> existingTagsValues = existingTags.stream().map(Object::toString).collect(Collectors.toList());
         return Splitter.on(",").trimResults().omitEmptyStrings().splitToList(tagsExpression).stream()
-                .map(this::toCucumberTag).filter(t -> !existingTagsValues.contains(t)).collect(Collectors.toList());
+                .map(CucumberWithSerenity::toCucumberTag).filter(t -> !existingTagsValues.contains(t)).collect(Collectors.toList());
     }
 
-    private String toCucumberTag(String from) {
+    private static String toCucumberTag(String from) {
         String tag = from.replaceAll(":","=");
         if (tag.startsWith("~@") || tag.startsWith("@")) { return tag; }
```

Comment:
Isn't '~' deprecated now in favour of 'not', according to the [tag expressions documentation](https://github.com/cucumber/cucumber/tree/master/tag-expressions)?